### ISSUE TITLE
Cannot enable Bluetooth (Catalina)

### ISSUE THREAD

**Issue by ApexHog** [gnodipac886/MatebookXPro-hackintosh#42]
I upgraded to Catalina using the latest update provided here (awesome, thank you).  But I cannot get the Bluetooth to work.  I have followed the VMware/TinyLinuxCore method but when I try to enable the Intel Bluetooth Adapter (select check box), I get a message: 

**'The USB device "Intel Bluetooth Adapter" is unable to connect to the guest due to a host application or driver claiming the device.'** 

I've checked that VMware has the right permissions etc and various combinations of booting (warm and cold) but no luck.  Any suggestions appreciated...

**Comment by byronpc** [comment]
hey, just wondering...did you upgrade to catalina from the app store (from mojave?)

**Comment by ApexHog** [comment]
Yes, I did.

**Comment by AshleyMiller9696** [comment]
Can confirm, doesn't work for me either

**Comment by ApexHog** [comment]
Apple TV doesn't work for me either :(

**Comment by byronpc** [comment]
so it was previously working on mojave on your machines. just stopped working when you upgraded to catalina right? i haven't personally upgraded yet. have you tried updating all kexts, (especially virtualsmc?)

**Comment by AshleyMiller9696** [comment]
I just started a few days with Catalina/Hackintosh
Copied everything from Clover to /Clover

**Comment by ApexHog** [comment]
I didn’t try BT until after I had upgraded to Catalina - so not sure if it worked or not in Mojave.

**Comment by ApexHog** [comment]
Ok, so updated kexts didn't change anything, unfortunately.  Doing the tce-load and then selecting the checkbox definitely does something but it's not enabling the BT.

**Comment by AshleyMiller9696** [comment]
What did you change?
Because I tried it with windows and that disables my bluetooth

**Comment by ApexHog** [comment]
> What did you change?

I updated the kexts.  I tried using Tinycorelinux to enable BT but that failed as before.  Then restarted into Windows.  BT did not show up at all there.  Did a complete shutdown and restart, BT was then working in Windows.  Then did a restart to Catalina and hey presto, BT working.


Kext | Version
-- | --
AppleALC | 1.4.3
CodecCommander | 2.7.1
CPUFriend | 1.1.9
HibernationFixup | 1.3.0
Lilu	 | 1.3.9
NoTouchID | 	1.0.2
NullEthernet | 	1.0.6
USBInjectAll | 	0.7.1
VirtualSMC | 	1.0.9
VoodooPS2 | 	2.0.4
WhateverGreen | 	1.3.4
